docs/deployment/add-ons/cloudwatch/README.md

```diff
@@ -6,9 +6,9 @@ The EKS Cluster will need a IAM role asscoiated with one of its worker nodes wit
 export CLUSTER_NAME=<>
 export CLUSTER_REGION=<>
 export NODEGROUP_NAME=$(aws eks list-nodegroups --cluster-name $CLUSTER_NAME --region $CLUSTER_REGION | jq -r '.nodegroups[0]')
-export ROLE_NAME=$(aws eks describe-nodegroup --cluster-name $CLUSTER_NAME --nodegroup-name $NODEGROUP_NAME --region $CLUSTER_REGION --output json | jq -r '.nodegroup.nodeRole' | grep -o '[^/]*$')
+export NODE_ROLE_NAME=$(aws eks describe-nodegroup --cluster-name $CLUSTER_NAME --nodegroup-name $NODEGROUP_NAME --region $CLUSTER_REGION --output json | jq -r '.nodegroup.nodeRole' | grep -o '[^/]*$')
```

Comment:
why are we adding permission to nodegroup role instead of using IRSA, is this coming from EKS docs?

Reply:
There is option to use https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-prerequisites.html iam service account, I tried this and it didnt work. Looked it up on github and others verified that it didn't seem to work, in the end the user can install this however the want and all of this documentation is redundant not sure how much re-treading we want.